File: accommodation_service/src/main/java/huy/project/accommodation_service/core/domain/dto/request/CreateLocationDto.java
package huy.project.accommodation_service.core.domain.dto.request;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class CreateLocationDto {
    private Long countryId;
    private Long provinceId;
    private Double longitude;
    private Double latitude;
    private String detailAddress;
}

File: location_service/LocationService.Core/Domain/Dto/Request/CreateLocationDto.cs
using LocationService.Core.Domain.Entity;

namespace LocationService.Core.Domain.Dto.Request
{
    public class CreateLocationDto
    {
        public long CountryId { get; set; }
        public long ProvinceId { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string Detail { get; set; }
        public string Name { get; set; }

        public LocationEntity ToEntity()
        {
            return new LocationEntity
            {
                CountryId = CountryId,
                ProvinceId = ProvinceId,
                Latitude = Latitude,
                Longitude = Longitude,
                Detail = Detail,
                Name = Name
            };
        }
    }
}